src/libraries/Common/src/Interop/Windows/BCrypt/Cng.cs

```diff
@@ -12,6 +12,7 @@
 using static Interop.BCrypt;
 using Microsoft.Win32.SafeHandles;
 
+#nullable enable
```

Comment:
This is needed because some other System.*.dll projects that haven't yet been annotated include this file?  Let's make sure to follow-up to delete these once they're unnecessary.

Reply:
Do you prefer having new issue for tracking or add another checkbox at the end of https://github.com/dotnet/corefx/issues/40623

Reply:
Either is fine.

Reply:
Added a checkbox to the main issue dotnet/corefx#40623